### ISSUE TITLE
[Bug]: ReferenceError in 8.0.0-beta.5 when using `docgen: "vue-component-meta"`

### ISSUE THREAD

**Issue by Clarkkkk** [storybookjs/storybook#26179]
### Describe the bug
Hi, I'm trying out the new docgen option `vue-component-meta`, and have an error.

Here is the reproducion [repo](https://github.com/Clarkkkk/Netease-Music-App).
```
pnpm install
pnpm storybook
```
open `http://localhost:6006/?path=/docs/%E5%85%AC%E5%85%B1%E7%BB%84%E4%BB%B6-drawer-drawer--docs` to see the error.
```
ReferenceError: createContext is not defined
    at index.ts:7:15
```
The error is gone when switch back to `vue-docgen-api`.

### To Reproduce

https://github.com/Clarkkkk/Netease-Music-App

**Comment by vanessayuenn** [comment]
@chakAs3 @larsrickert @kasperpeulen any idea what's happening here?

**Comment by larsrickert** [comment]
The issue might be that array destructuring is used for the export. I don't thing we have tested for this use case.
@Clarkkkk Does it work when you change it to:

```ts
```

**Comment by Clarkkkk** [comment]
> The issue might be that array destructuring is used for the [export](https://github.com/Clarkkkk/Netease-Music-App/blob/main/src/components/Drawer/context.ts#L10C1-L10C90). I don't thing we have tested for this use case. @Clarkkkk Does it work when you change it to:
> 
> ```ts
> const context = createContext<DrawerContext>('drawer-context');
> 
> export const provide = context[0];
> export const useDrawerContext = context[1];
> ```

I tried it just now, and unfortunately the error remains.

**Comment by larsrickert** [comment]
@Clarkkkk @vanessayuenn I fixed the bug in #26303 :)

**Comment by Clarkkkk** [comment]
> @Clarkkkk @vanessayuenn I fixed the bug in #26303 :)

Thank you! Can confirm by patching your fix!
